clients/src/main/java/org/apache/kafka/clients/consumer/internals/PartitionAssignor.java

```diff
@@ -261,4 +261,47 @@ public String toString() {
         }
     }
 
+    class MemberInfo implements Comparable<MemberInfo> {
```

Comment:
I think we can put it in `AbstractPartitionAssignor` which is an internal class and being extended by all built-in assignors. Although keeping it here with default access also prevents users to get to it, I felt it was a bit less elegant.

Reply:
That sgtm!